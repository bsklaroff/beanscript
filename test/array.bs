arr = [23, 0, -47]
@print(arr[0])
@print(arr[1])
@print(arr[2])
nested_arr = [[], [5, 10], arr]
arr[1] = arr[0] * arr[2]
@print(arr[0])
@print(arr[1])
@print(arr[2])
@print(nested_arr[1][0])
@print(nested_arr[1][1])
@print(nested_arr[2][0])
@print(nested_arr[2][1])
@print(nested_arr[2][2])
nested_arr[2][0] = nested_arr[1][0] + 7
@print(arr[0])
@print(arr[1])
@print(arr[2])
@print(nested_arr[1][0])
@print(nested_arr[1][1])
@print(nested_arr[2][0])
@print(nested_arr[2][1])
@print(nested_arr[2][2])
