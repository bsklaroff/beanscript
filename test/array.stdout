23
0
-47
23
-1081
-47
5
10
23
-1081
-47
12
-1081
-47
5
10
12
-1081
-47
3
3
